### PR TITLE
Added Aug Community Newsletter

### PR DESCRIPTION
**Pull request by raukadah**
Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

File: content/2020/August/community_news.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: "PythonPune Community Newsletter [August 2020 Edition]"
+date: 2020-08-30T00:03:41+05:30
+url: newsletter/july-2020
```
Comment (bhavin192) [inline content/2020/August/community_news.md]:
```suggestion
url: newsletter/august-2020
```

File: content/2020/August/community_news.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  
+* Interactively grep source code. Source 
+  https://github.com/livegrep/livegrep
+ 
+* Run only the tests that are relevant for your changes 
+  https://github.com/man-group/partialtesting
+
+## Upcoming events in & around Pune
+
+* Tickets for PyCon India 2020 is now open!
```
Comment (bhavin192) [inline content/2020/August/community_news.md:76]:
```suggestion
* Tickets for PyCon India 2020 are available now! 
```

File: content/2020/August/community_news.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+* pytest major version lands 6.0 
+  https://docs.pytest.org/en/stable/changelog.html
+
+## New projects
+* A Git-based tool for comparing branches, designed to work with a rebase-oriented Git workflow. 
+  https://github.com/MarkForged/GOMP
+ 
+* Rich is a Python library for rich text and beautiful formatting in the terminal. 
+  https://github.com/willmcgugan/rich -
```
Comment (bhavin192) [inline content/2020/August/community_news.md:56]:
```suggestion
  https://github.com/willmcgugan/rich
```

**Review by bhavin192 — APPROVED** [review]
Looks good!

**Review by raukadah — COMMENTED** [review]
Suggestions added!

**Pull request merged**